### PR TITLE
figlet: add page

### PR DESCRIPTION
**Pull request by sweeper3000**

### CODE REVIEW

File: pages/linux/figlet.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# figlet
+
+> Generate ASCII banners from user input.
+
+- Generate by directly inputting text:
+
+`figlet {{some_text_you_write}}`
```
Comment (agnivade) [inline pages/linux/figlet.md]: Suggestion -> Change "some_text_you_write" to "input_text"

File: pages/linux/figlet.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Use a different font file:
+
+`figlet {{some_text_you_write}} -f {{font_file_name}}`
```
Comment (agnivade) [inline pages/linux/figlet.md]: same as above

**Comment by sweeper3000** [comment]
I'll fix the Travis build in a bit

File: pages/linux/figlet.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# figlet
+
+> Generate ASCII banners from user input.
+
+- Generate by directly inputting text:
+
+`figlet {{input_text}}`
+
+- Use a different font file:
```
Comment (waldyrious) [inline pages/linux/figlet.md]: I'd rather say "custom" instead of "different".

**Comment by waldyrious** [comment]
@sweeper3000 thanks for the PR! I think it is already mergeable, but if you don't mind, please address the comment I left inline.

**Pull request merged**